packages/react/src/components/Forms/Radio/index.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { InputHTMLAttributes, PropsWithChildren } from "react";
+import classNames from "classnames";
+import { Field, FieldProps } from "components/Forms/Field";
+
+type Props = InputHTMLAttributes<HTMLInputElement> &
+  FieldProps & {
+    label?: string;
+  };
+
+export const Radio = ({ label, text, state, ...props }: Props) => {
+  return (
+    <label
+      className={classNames("c__checkbox", "c__radio", {
+        "c__checkbox--disabled": props.disabled,
+      })}
+    >
+      <Field text={text} compact={true} state={state}>
+        <div className="c__checkbox__container">
+          <input type="radio" {...props} />
+          {label && <div className="c__checkbox__label">{label}</div>}
```

Comment:
Ditto. We should use a label element and reverse input and label order in the DOM.

Reply:
ditto'